lib/gooddata/models/project.rb

```diff
@@ -837,6 +837,17 @@ def execute_maql(maql, options = {})
       result
     end
 
+    # Executes MAQL expression
+    #
+    # @param maql [String] MAQL expression
+    # @return [String] URI to poll
+    def execute_maql_async(maql, options = {})
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.